src/Spatial/Euclidean/Circle.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MathNet.Spatial.Euclidean
+{
+    /// <summary>
+    /// Describes a standard 2 dimentional circle
+    /// </summary>
+    public class Circle : IEquatable<Circle>
+    {
+        /// <summary>
+        /// The center point of the circle
+        /// </summary>
+        public Point2D Center { get; }
```

Comment:
Minor style nag: we use style cop conventions meaning ctor before properties. Not very important but makes future diffs cleaner.

Reply:
ok, will update styles.  I didn't use readonly public fields deliberately in order to be able to later add support for lazy calculation where multiple construction methods exist